torch/csrc/jit/codegen/cuda/ops/normalization.cpp

```diff
@@ -69,6 +69,64 @@ TensorView* variance(
   return y;
 }
 
+TORCH_CUDA_CU_API VarMeanResult variance_mean(
+    TensorView* x,
+    const std::vector<int>& dims,
+    int64_t correction,
+    bool keepdim) {
+  TORCH_INTERNAL_ASSERT(x != nullptr, "Input is invalid.");
+
+  TORCH_CHECK(
+      correction >= 0, "correction must be non-negative, but got ", correction);
+
+  // There are compilation errors for half precision
```

Comment:
Should we consider just promote math to float for reduced precision then? that seems to be an universal way used by all others computing variance.

Reply:
I handle type promotion on the Python side.